library/analytics/src/main/java/org/cru/godtools/analytics/model/OpenToolsAnalyticActionEvent.kt

```diff
@@ -0,0 +1,10 @@
+package org.cru.godtools.analytics.model
+
+sealed class OpenToolsAnalyticActionEvent(action: String) : AnalyticsActionEvent(action) {
```

Comment:
should be named `OpenToolAnalyticsActionEvent`. my hunch is this should also be relocated to `:ui:base-tool` because that is where you actually should be triggering the event from.